authors/settings.py

```diff
@@ -134,10 +134,8 @@
 AUTH_USER_MODEL = 'authentication.User'
 
 REST_FRAMEWORK = {
-    'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
-    'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
```

Comment:
I notice the default authentication classes were changed, is there a particular reason why this was necessary? Please expound

Reply:
The authentication class that is used is not set up. I have however commented the line if there is a need to use it while working on authentication.